src/Persistence/Sql.php

```diff
@@ -776,16 +776,15 @@ public function loadAny(Model $model): array
 
     /**
      * Inserts record in database and returns new record ID.
-     *
-     * @return mixed
      */
-    public function insert(Model $model, array $data)
+    public function insert(Model $model, array $data): string
```

Comment:
Why return type is string? It can be integer too isn't it?

Reply:
![image](https://user-images.githubusercontent.com/2228672/92409418-d4495c80-f140-11ea-8500-34673a7391d8.png)